stream-api/src/main/java/com/bobocode/StreamTask_Collecting.java

```diff
@@ -14,15 +22,53 @@ public static void main(String[] args) {
         List<Account> accounts = TestDataProvider.generateAccountList();
 
         // 1 - Find a set of account owner's names(first name) that have a length <= 4
+        System.out.println("Find a set of account owner's names(first name) that have a length <= 4");
```

Comment:
It's just a refinement. If you want to fix it, you can change the code, commit it, and then push to the server.